dev/VSIX/Directory.Build.props

```diff
@@ -4,7 +4,8 @@
     <!-- Provide default nuget feed (Windows App SDK internal) and package versions for developer builds -->
     <PropertyGroup>
         <RestoreSources Condition="'$(RestoreSources)'==''">
```

Comment:
is this redundant with nuget.config? necessary?

Reply:
I think you can remove RestoreSources - it's C#-specific (so nonstandard)

Reply:
I can do that. I'm not sure why it is here.